src/main/java/com/refinedmods/refinedstorage/datageneration/RecipeGenerator.java

```diff
@@ -26,7 +29,7 @@ protected void registerRecipes(Consumer<IFinishedRecipe> consumer) {
                     .addIngredient(tag)
                     .addIngredient(color.getTag())
                     .setGroup(RS.ID)
-                    .addCriterion("cobblestone", InventoryChangeTrigger.Instance.forItems(Blocks.COBBLESTONE))
+                    .addCriterion("refinedstorage:color", InventoryChangeTrigger.Instance.forItems(RSItems.CONTROLLER.get(DyeColor.LIGHT_BLUE).get()))
```

Comment:
Shouldn't this be named `refinedstorage:controller`

Reply:
yes it should be.